code/wdb/datastripe.q

```diff
@@ -42,12 +40,15 @@ initdatastripe:{
     // update endofperiod function
     endofperiod::.wdb.datastripeendofperiod;
     
-    .wdb.tablekeycols:.ds.loadtablekeycols[];  // replace with dictionaries from json file
+    // load in variables
+    .wdb.tablekeycols:.ds.loadtablekeycols[];
     t:tables[`.] except .wdb.ignorelist;
+
+    // create or load the access table
```

Comment:
```suggestion
    // load the access table, fall back to generating table if load fails
```
think something like this would be more descriptive of what's happening here